src/providers/authentication/authentication.ts

```diff
@@ -101,6 +101,13 @@ export class AuthenticationProvider {
   }
 
   public getAuthenticationToken = async (): Promise<string> => {
+
+    // @TODO remove temporary solution which checks if token has expired manually then forces token expiry
```

Comment:
A JIRA ticket number should be assigned to this TODO

Reply:
@szabikr - added jira ticket

Reply:
awesome, thanks @andrewsetterfield